src/components/Header.tsx

```diff
@@ -28,13 +46,51 @@ const useStyles = createUseStyles({
 		display: 'flex',
 		alignItems: 'center',
 	},
+	submitButton: {
+		...button,
+		background: '#ffa666',
+		'&:hover': {
+			background: '#ffb37c',
+		},
+		'&:active': {
+			background: '#ffc093',
+		},
+	},
+	disableButton: {
+		...button,
+		background: '#979797',
+	},
+	badge: {
+		height: '15px',
+		width: '15px',
+		borderRadius: '50%',
+		color: '#fff',
+		border: 'none',
+		boxShadow: '0 2px 5px rgba(0, 0, 0, 0.15)',
+		fontWeight: '600',
+		fontSize: '12px',
+		lineHeight: '15px',
+		position: 'absolute',
+		top: '-5px',
+		right: '-5px',
+		background: '#ed4300',
+	},
+	disableBadge: {
+		display: 'none',
+	},
 });
 
 function Header() {
 	const schemaStore = useSchemaStore();
 
 	const classes = useStyles();
 
+	const requestsExist = computed(() => schemaStore.requestsStore.preparedRequests.length > 0).get();
+
+	const submitChanges = () => {
+		schemaStore.requestsStore.saveChanges()
```

Comment:
we can just pass schemaStore.requestsStore.saveChanges to onClick button handler